io_context.go

```diff
@@ -3,11 +3,16 @@ package astiav
 //#cgo pkg-config: libavformat
 //#include <libavformat/avformat.h>
 import "C"
-import "unsafe"
+import (
+	"io"
+	"unsafe"
+)
 
 // https://github.com/FFmpeg/FFmpeg/blob/n5.0/libavformat/avio.h#L161
 type IOContext struct {
 	c *C.struct_AVIOContext
+	io.ReadWriteCloser
```

Comment:
Can you remove those 2 lines?

Reply:
Actually i wanted to use IOContext as an io.Writer / io.Reader.

So i can pass data in memory directly to it.

Reply:
Sure, but there's no need to declare interfaces here, the simple fact that IOContext would implement those interfaces is sufficient